utils/config/daos_server.yml

```diff
@@ -163,6 +163,12 @@
 #control_log_file: /tmp/daos_control.log
 #
 #
+## Enable daos_admin (privileged helper) logging.
+#
+## default: disabled (errors only to control plane log)
+#helper_log_file: /tmp/daos_admin.log
```

Comment:
Do the example config files need to be updated, too?

Reply:
Yeah, I considered that, but those configs don't list all available options. We probably don't want to recommend that this option be set in production because it enables DEBUG-level logging.

Reply:
OK, that's fair.